src/components/chrome/chrome.tsx

```diff
@@ -48,118 +42,323 @@ if (localStorage.getItem('theme') === 'dark') {
+            listItems={alterLinksWithCurrentState(LearnLinks)}
+            pinTitle={addLinkNameToPinTitle}
+            onPinClick={addPin}
+            maxWidth="none"
+            color="subdued"
+            gutterSize="none"
+            size="s"
+          />
+        </EuiCollapsibleNavGroup>
+
+        {/* Docking button only for larger screens that can support it*/}
+        <EuiShowFor sizes={['l', 'xl']}>
+          <EuiCollapsibleNavGroup>
+            <EuiListGroupItem
+              size="xs"
+              color="subdued"
+              label={`${navIsDocked ? 'Undock' : 'Dock'} navigation`}
+              onClick={() => {
+                setNavIsDocked(!navIsDocked);
+                localStorage.setItem(
+                  'navIsDocked',
+                  JSON.stringify(!navIsDocked)
+                );
+              }}
+              iconType={navIsDocked ? 'lock' : 'lockOpen'}
+            />
+          </EuiCollapsibleNavGroup>
+        </EuiShowFor>
+      </EuiFlexItem>
+    </EuiCollapsibleNav>
+  );
+
+  const headers = (
+    <>
+      <EuiHeader
+        theme="dark"
+        position="fixed"
+        sections={[
+          {
+            items: [
+              <EuiHeaderLogo iconType="logoElastic">Elastic</EuiHeaderLogo>,
+            ],
+            borders: 'none',
+          },
+          {
+            items: [
+              <EuiPopover
+                button={
+                  <EuiHeaderSectionItemButton
+                    aria-label="Account menu"
+                    onClick={() => setIsAccountOpen(!isAccountOpen)}>
+                    <EuiAvatar name="John Username" size="s" />
+                  </EuiHeaderSectionItemButton>
+                }
+                isOpen={isAccountOpen}
+                closePopover={() => setIsAccountOpen(false)}>
+                <div>
+                  <EuiButton
+                    size="s"
+                    iconType="invert"
+                    onClick={() =>
+                      setTheme(theme === 'light' ? 'dark' : 'light')
+                    }
+                    isLoading={themeIsLoading}>
+                    Switch Theme
+                  </EuiButton>
+                </div>
+              </EuiPopover>,
+            ],
+            borders: 'none',
+          },
+        ]}
+      />
+      <EuiHeader
+        position="fixed"
+        sections={[
+          {
+            items: [
+              collapsibleNav,
+              <EuiPopover
+                button={
+                  <EuiHeaderSectionItemButton
+                    aria-label="Account menu"
+                    onClick={() => setIsSpacesOpen(!isAccountOpen)}>
+                    <EuiAvatar type="space" name="Default Space" size="s" />
+                  </EuiHeaderSectionItemButton>
+                }
+                isOpen={isSpacesOpen}
+                closePopover={() => setIsSpacesOpen(false)}>
+                <div>asdf</div>
```

Comment:
The popover could have a better text.  Or if the goal is to represent an account menu, maybe a few fake links as an example.

Reply:
Good catch, fixed

![](https://snid.es/2020JUL/531S-MdSSmDIGMqg.gif)